src/modules/Dropdown/Dropdown.js

```diff
@@ -747,7 +802,13 @@ export default class Dropdown extends Component {
         active={isActive(opt.value)}
         onClick={this.handleItemClick}
         selected={selectedIndex === i}
-        text={opt.text}
+        text={
+          allowAdditions
+          && (additionPosition === 'top' ? i === 0 : i === options.length - 1)
+          && opt.text.startsWith('Add: ') ?
+          <span><strong><em>Add:</em></strong> {opt.text.substring(5)}</span> :
+          opt.text
+        }
```

Comment:
I like the look of it provides, but here are my concerns:

1. It fails `DropdownItem` `text` prop validation, unless we need to allow nodes on `text` here. However doing so would go against our current shorthand prop standards (string and numbers only) #391.
1. There is a lot of brittle logic here which I'm not sure justifies the gain (bold italic).  There are also a lot of tests required to test all the code branches.
1. I think this is better suited for core SUI CSS since it is purely stylistic.

So again, I'm not opposed to this feature and I agree the look it gives far better than without it.  But, I don't think the required tradeoffs are worth the bold/italic styling in this case.

I'm open to any other ideas or counter arguments before making a final call.  Let me know what you think.